File: lang/php/src/muse/algorithms/number_theory/Parity.php
<?php
    namespace muse\algorithms\number_theory\Parity;

    function moduloIsEven(int $n): bool {
        return $n % 2 == 0;
    }

    function moduloIsOdd(int $n): bool {
        return $n % 2 != 0;
    }

    function bitwiseIsEven(int $n): bool {
        return ($n & 1) == 0;
    }

    function bitwiseIsOdd(int $n): bool {
        return ($n & 1) != 0;
    }
?>

File: go/muse/algorithms/number_theory/parity/parity.go
package parity

func ModuloIsEven(n int64) bool {
	return n%2 == 0
}

func ModuloIsOdd(n int64) bool {
	return n%2 != 0
}

func BitwiseIsEven(n int64) bool {
	return (n & 1) == 0
}

func BitwiseIsOdd(n int64) bool {
	return (n & 1) != 0
}
